tests/unit-tests/13-modules/#.scm

```diff
@@ -38,9 +64,9 @@
                                `(namespace (,namespace2)))
                            `(##import ,namespace2)))
                       " returned "
-                      (object->string behavior2)
+                      (behavior->string behavior2)
                       " but expected "
-                      (object->string behavior1)))))
+                      (behaviour->string behavior1)))))
```

Comment:
`behavior`...

Reply:
I will use `object->string` directly. This change is not pushed yet.